src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5074,4 +5074,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_VoidInTuple" xml:space="preserve">
     <value>A tuple may not contain a value of type 'void'.</value>
   </data>
-</root>
+  <data name="ERR_NonTaskMainCantBeAsync" xml:space="preserve">
+    <value>Async Main methods must return Task or Task&lt;int&gt;</value>
```

Comment:
Not technically, but I think this is fine for an error message.

I'd be willing to change it, but "Async Main methods must return either Task or Task<T> whereby the return type of GetAwaiter().GetResult() is either `void` or `int`" seems a bit verbose for an error message.